lib/danger/request_sources/github.rb

```diff
@@ -14,7 +14,11 @@ def api_url
     end
 
     def fetch_details
-      response = REST.get api_url
+      base_api_token = Base64.strict_encode64(ENV["DANGER_API_TOKEN"])
+      response = REST.get api_url, {}, {
+        'Authorization' => "Basic #{ base_api_token }",
```

Comment:
Space inside string interpolation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.